### ISSUE TITLE
Add Openmod to documentation

### ISSUE THREAD

**Issue by rube200** [SmartlyDressedGames/U3-Docs#32]
there is a vast amount of documentation for openmod, and slowly the hosts will move from rocketmod to openmod, so I suggest that openmod be mentioned / shown in the unturned documentation such as the url of om documentation.

**Comment by SDGNelson** [comment]
If it does naturally become popular I will include it in the official docs, but I do not want to prematurely promote further fragmentation of the modding APIs. One benefit of OpenMod is the nice API wrapping some of the legacy mess, which would ideally get cleaned up on my end, but is hard to justify working on.

One step I do think would make sense is integrating the plugins list for OpenMod in the server info screen, and the server filters.

Curious to hear if Trojaner thoughts on this stance. So far I get the impression that he is steadily working on it, and views this early promotion as overeager.

**Comment by Trojaner** [comment]
At the moment I have not created or requested such additions on purpose, as the OpenMod API is currently lacking general game abstractions and has no wrappers around Unturned code like RocketMod did. It is currently just a generic plugin framework which does not have any games specific concepts / abstractions, plugins must currently call Unturned code directly. Aside this there are also some breaking bugs such as not supporting console input through Windows APIs, which has been reported by game hosters and prevents them from adaopting it.

With the upcoming 2.0 we are implementing [generic game abstractions](https://github.com/openmod/openmod/tree/master/extensions/OpenMod.Extensions.Games.Abstractions) for stuff like players, inventory, vehicles, entities, etc. that would make universal plugins possible with many games.  After that version 3 will focus on Unturned wrappers, utilities, events, etc. 

All big plugin vendors have already experimented with OpenMod. ImperialPlugins, RedstonePlugins and AviRockets are planning on migrating to it.

In short, I would like to wait until OpenMod matures a bit before wider adoption happens.

**Comment by SDGNelson** [comment]
Thanks for the clarification Trojaner.
